src/git/git_integration_test.go

```diff
@@ -346,3 +347,20 @@ func TestNewGitGetCurrentVersionFromRepoWithGreatNumbersTag(t *testing.T) {
 	tests.AssertEqualValues(t, "3.0.2", result)
 	f.cleanLocalRepo(t)
 }
+
+func TestNewGitGetCurrentVersionFromRepoWithDisorderedTags(t *testing.T) {
```

Comment:
@julianogalgaro 

This is the test I've added to cover disordered tag numbers.

Reply:
Nice job!

Reply:
Now you can mark as done the requirement:

“I have added tests that prove my fix is effective or that my feature works“